wordpress/wp-content/plugins/shockingly-simple-favicon/shockingly-simple-favicon.php

```diff
@@ -62,8 +62,8 @@ function favi_deactivate() {
 function favi_geturl() {
 	$opt = get_option('favi_options');
 	$icon = $opt['url'];
-	if ( $icon == '' || $icon == NULL ) { // se favicon url n�o setada, seta como favicon padr�o
-		$icon = get_settings('siteurl') . '/wp-content/plugins/shockingly-simple-favicon/default/favicon.ico';
+	if ( $icon == '' || $icon == NULL ) { // se favicon url n�o setada, seta como favicon padr�o

```

Comment:
File encoding changed, from ISO-8859-1 to UTF-8. Technically this isn't a problem since it should be UTF-8, and because this is a comment, not something the end user will see.

Reply:
oh wow, you've got a really good eye. thank you, i will note it and look out for it, in case of some embedded literals